zebra/zebra_nhg.c

```diff
@@ -0,0 +1,514 @@
+/* Zebra NHG Code.
+ * Copyright (C) 2019 Cumulus Networks, Inc.
+ *                    Donald Sharp
+ *                    Stephen Worley
+ *
+ * This file is part of FRR.
+ *
+ * FRR is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * FRR is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FRR; see the file COPYING.  If not, write to the Free
+ * Software Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+ * 02111-1307, USA.
+ */
+#include "zebra.h"
+
+#include "lib/nexthop.h"
+#include "lib/nexthop_group.h"
+#include "lib/jhash.h"
```

Comment:
shouldn't `zebra.h` be `lib/zebra.h`, to match any other headers from `lib` ?

a number of the headers don't seem to be needed: nexthop, nexthop_group, jhash, and zserv?

Reply:
Yep, those are unneeded. I think they got carried over from another commit.

As to `lib/zebra.h`, there is a discrepancy because some places use that format and other use `<zebra.h>`. Either way this commit is wrong. Which one should we prefer moving forward?